firefox_puppeteer/tests/test_utils.py

```diff
@@ -0,0 +1,49 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness import FirefoxTestCase
+
+
+class TestSanitize(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        # Clear all previous history and cookies.
+        self.places.remove_all_history()
+        self.marionette.delete_all_cookies()
+
+        urls = [
+            'layout/mozilla_projects.html',
+            'layout/mozilla.html',
+            'layout/mozilla_mission.html',
+            'cookies/cookie_single.html'
+        ]
+        urls = [self.marionette.absolute_url(t) for t in urls]
+
+        # Open the test urls, including the single cookie setting page.
+        def load_urls():
+            with self.marionette.using_context('content'):
+                for url in urls:
+                    self.marionette.navigate(url)
+        self.places.wait_for_visited(urls, load_urls)
+        self.assertEqual(self.places.get_all_urls_in_history(), urls)
+        self.assertIsNotNone(self.marionette.get_cookie('litmus_1'))
```

Comment:
If I move the URLs in history assert statement to `test_sanitize_history` then it will not have access to the list `urls` which is a local variable of the `setup()` method and the assert statement in `test_sanitize_history` will fail.  Should I make `urls` in `setup()` an instance attribute, i.e. `self.urls = [..]` and then this will be accessible in the test method?